### PR TITLE
chore: update to latest Angular Framework and CLI 8.x betas, TypeScript 3.3

### LINKED ISSUE #1673: Build: Update to latest Angular, TypeScript, Node dependencies
As we move towards version 8, we need to bump these to the latest dependencies and fix any relevant errors.

- Angular: Latest Beta
- TypeScript: 3.3333... or 3.4?
- Node: 11

### PR DESCRIPTION
**Pull request by brandonroberts**
Closes #1673

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[x] Build related changes
[x] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

```
[x] Yes
[ ] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -77,31 +77,31 @@
     ]
   },
   "dependencies": {
-    "@angular/animations": "^7.0.1",
+    "@angular/animations": "^8.0.0-beta.10",
```
Comment (timdeschryver) [inline package.json]: Do we also have to update the package version number?

File: tools/defaults.bzl [hunk 2]
```diff
@@ -9,8 +9,8 @@ load("@npm_angular_bazel//:index.bzl", _ng_module = "ng_module", _ng_package = "
 load("@npm_bazel_typescript//:defs.bzl", _ts_library = "ts_library")
 
 DEFAULT_TSCONFIG = "//:tsconfig.json"
-NG_VERSION = "^7.0.0"
-RXJS_VERSION = "^6.0.0"
+NG_VERSION = "^8.0.0-beta"
```
Comment (timdeschryver) [inline tools/defaults.bzl]: We also have to update `platformVersion` at https://github.com/ngrx/platform/blob/master/modules/schematics-core/utility/libs-version.ts#L1

File: package.json (hunk 1 above)
Comment (brandonroberts) [reply package.json]: Not sure what you mean?

Comment (timdeschryver) [reply package.json]: Sorry, I meant this line https://github.com/ngrx/platform/blob/master/package.json#L3

**Pull request merged**